docs/reference/README.md

```diff
@@ -0,0 +1,23 @@
+## jsctl
```

Comment:
This file seems to be the same as docs/reference/jsctl.md - is that expected?

Reply:
Thanks for spotting that I removed that one